components/gfx/display_list/mod.rs

```diff
@@ -976,6 +985,7 @@ pub enum DisplayItem {
     StackingContextClass(Arc<StackingContext>),
     LayeredItemClass(Box<LayeredItem>),
     NoopClass(Box<BaseDisplayItem>),
+    IframeClass(Box<IframeDisplayItem>),
```

Comment:
It feels like NoopClass could be removed in favor of the new IframeClass. NoopClass exits just as a placeholder so that iframes can hold a place in the display list without drawing anything.